imports/api/locations/locations.js

```diff
@@ -38,7 +38,8 @@ Schema.Locations = new SimpleSchema({
     max: 180
```

Comment:
we've decided not to store affordances in the location db, there's no need to since we only need them on a location change